teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './person'
+
+class Teacher < Person
+  attr_reader :age, :name, :specialization
+
```

Comment:
- Hey, since the teacher inherits from the parent class, you don't need the setter and getter method for it. please, kindly remove this.